administration/configuring-fluent-bit/yaml/configuration-file.md

```diff
@@ -81,6 +81,7 @@ The `service` section defines the global properties of the service. The Service
 | `scheduler.cap`   | Set a maximum retry time in seconds. Supported from v1.8.7. | `2000` |
 | `scheduler.base`  | Sets the base of exponential backoff. Supported from v1.8.7. | `5` |
 | `json.convert_nan_to_null` | If enabled, NaN is converted to null when Fluent Bit converts `msgpack` to JSON. | `false` |
+| `json.escape_unicode` | When Fluent Bit is serializing a MessagePack record into JSON, this option allows to specify the encoding of raw UTF-8 strings. The available options are `escaped` and `raw`. `escaped` is the default and it will escape the UTF-8 characters. `raw` will not escape the UTF-8 characters. | `escaped` |
```

Comment:
`json.escape_unicode` handles boolean value. So,we need to describe to use with `On`/`true` or `Off`/`false`.